docs/explanations/lifecycle.md

```diff
@@ -6,7 +6,7 @@ of being written, loaded and run. Take the following plan.
     from typing import Any, List, Mapping, Optional, Union
     
     import bluesky.plans as bp
-    from bluesky.utils import MsgGenerator
+    from blueapi.core import MsgGenerator
```

Comment:
```suggestion
    from bluesky.utils import MsgGenerator
```

Reply:
revert

Reply:
ok but why? otherwise I got the error, I spent 30 minutes on this yesterday

Reply:
As discussed on slack, intervening change fixed compatibility to bluesky>=1.13